stroom-query/stroom-query-api/src/main/java/stroom/datasource/api/v2/TextField.java

```diff
@@ -62,4 +62,23 @@ public TextField(@JsonProperty("name") final String name,
     public String getType() {
         return FieldTypes.TEXT;
     }
+
+    @Override
+    public String getShortTypeName() {
+        return "text";
+    }
+
+    @Override
+    public String getTypeDescription() {
+        return "Text field type\n" +
```

Comment:
If the IDE will auto convert them I would change all these big description block to java text blocks to avoid all the escaping and \n use.

Reply:
Text blocks aren't supported by GWT. Will leave as-is.